### ISSUE TITLE
64-bit

### ISSUE THREAD

**Issue by persianpros** [RaspberryPLi/PLiPi#1]
@MastaG As you can see in https://github.com/RaspberryPLi/meta-raspberrypi/blob/morty/README.md we're using kernel 4.4.x and https://github.com/RaspberryPLi/meta-raspberrypi/blob/morty/conf/machine/raspberrypi3.conf for RPi3 which is 32-bit but I wan't to use https://github.com/RaspberryPLi/meta-raspberrypi/blob/morty/conf/machine/raspberrypi3-64.conf with kernel 4.9.x but I'm not sure PLi's enigma2 is ready for it or not.

I just know OE-A fixed 64-bit problems in their sources.

**Comment by MastaG** [comment]
Then we can easily track down their changes provide external patches for enigma2 in our BSP layer :)
I'll add the layer to my pli-oe-core.

**Comment by persianpros** [comment]
Yeah sure :) Build is ok now with morty (develop) branch but if you remove https://github.com/RaspberryPLi/meta-raspberrypi/blob/morty/recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_%25.bbappend and I don't know what's the problem right now ...

**Comment by persianpros** [comment]
Please make your membership public via https://github.com/orgs/RaspberryPLi/people :)

I had to do this https://github.com/RaspberryPLi/meta-raspberrypi/commit/bf43a363c9e0e0a04ed2f9a5054a1508451fb8dc as I don't have a solution for now ... (Please help me if you can)

RPi3 is super fast with PLi 6.x even in 32-bit mode just test it ;)

Mouse doesn't work but keyboard is ok, I think we need to use https://github.com/RaspberryPLi/enigma2/commit/bdee276e5aa38b17b8b68453d0f8bfe9f67c94c0 for the keyboard as it's not easy to work with RPi without Open WebIF remote, I did ask https://github.com/technic to help us.

Some 64-bit tweaks:
https://github.com/RaspberryPLi/enigma2/commit/66ad8b0f32953fe209655a54e3f9f097df6b24b1
https://github.com/RaspberryPLi/enigma2/commit/d26c4dda412dc1464a561b5f4aa38fea7de7688a

**Comment by persianpros** [comment]
64-Bit version crashes after boot but maybe without https://github.com/RaspberryPLi/meta-raspberrypi/commit/3bfb84cdc79f559f19b3c2b4fbaa062f1c8ae5b1 it would be fine, I have to test it.

enigma2 could not understand RPi's HDMI and detects it as YPbPr with no "Video mode" and no "Refresh rate".
